src/components/buttons/back-button/back-button.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Button } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { createBrowserHistory } from 'history';
+import buttonTitles from '../../../configs/button-titles';
+
+const BackButton = ({ type, variant, color, ...props }) => {
+  const { GO_BACK_TITLE } = buttonTitles;
+  const history = createBrowserHistory();
+  return (
+    <Button
+      type={type}
+      color={color}
+      variant={variant}
+      onClick={() => history.goBack()}
+      {...props}
```

Comment:
Its better to set `data-cy='back-btn'` here.

Reply:
fixed